src/transform/BodySpacingTransform.ts

```diff
@@ -0,0 +1,65 @@
+interface Spacing {
+  top?: string
+  right?: string
+  bottom?: string
+  left?: string
+}
+
+/**
+ * Sets the margins on the body element via inline styles.
+ * @param {!Document} document
+ * @param {Spacing} values { top, right, bottom, left }
+ *   Use value strings with units, e.g. '16px'. Undefined values are ignored.
+ * @param callback callback function
+ * @return {void}
+ */
+const setMargins = (document: Document, values: Spacing, callback?: () => void): void => {
+  const body: HTMLElement = document.body
```

Comment:
May consider just passing these the body element since document isn't used for anything else beyond getting the body...

Reply:
We could make this transform more generic but then it adds more burden to the caller. Ok, maybe not so much. I was actually on the fence about it. If you feel like we should then I'd probably rename this transform to not have `body` in the name anymore. Let me know.

Reply:
Oh, I think it's fine to have 'body' in the name and the 'document' param renamed to 'body'.

Reply:
Done